package exceptionTest02;
// 14번 : 메소드 선언부에서의 throws
public class ThrowsTest02 {
	public static void main(String[] args) {
		
	}
	
	static void method() {
		System.out.println("양수만 입력하세요!!");
	}
	
}
